TCAT/Utilities/Styles.swift

```diff
@@ -46,10 +74,35 @@ extension UIFont {
     
     /// Generate fonts for app usage
     static func style(_ name: String, size: CGFloat) -> UIFont {
+        var name = name
+        if size >= 14 {
+            switch name {
```

Comment:
don't switch on Strings b/c it assumes whoever uses the method will always input a valid font `name`, but if they don't, the return statement of `UIFont(name: name, size: size)!` will fail. 

`name` should be an enum and therefore `regular`, `medium`, `semibold`, and `bold` should be enums.

Also why is the `regular` font name called "regular"--is that the default system font or is it San Francisco?

Basically talking about an enum like this:
```
enum System: String {
    case regular, medium, semibold, bold
}
```
And then the `enum System` would be either within the `struct SanFrancisco`  if regular is a San Francisco font or within `struct Fonts` if regular is a default system font.

And then you can use the `rawValue` of the `enum` to get the `regular` string like:
```
let name = System.regular.rawValue // name is "regular"
```

Reply:
Great idea, thanks!